examples/universal_request_tracing/components/processor.py

```
@@ -0,0 +1,149 @@
+    """
+
+    def __init__(self):
+        self.prefiller_client: DynamoClient
+        self.decoder_client: DynamoClient
+        self.request_count = 0
+
+    @async_on_start
+    async def async_init(self):
+        """Asynchronous initialization method that runs on service startup."""
+        self.log("debug", "Initializing component clients")
+        self.prefiller_client, self.decoder_client = await asyncio.gather(
```

Ruff (0.11.9)

Undefined name `asyncio`

(F821)